apps/www/next.config.js

```diff
@@ -7,6 +7,11 @@ const { NODE_ENV, CDN_FRONTEND_BASE_URL } = process.env
 
 module.exports = withTM(
   withBundleAnalyzer({
+    compiler: {
+      reactRemoveProperties: {
+        properties: ['^data-test'],
```

Comment:
wouldn't that have to be env specific and left in when e.g. compiling with node env test?

Reply:
In the future yes (once we have E2E), as of now not necessarily.
But I can change it anyways. Should we check for TEST env-var to be defined?

Reply:
or remove it until we use it :P

Reply:
:( aight